setup.py

```diff
@@ -17,11 +17,20 @@
     test_suite="tests",
     install_requires=[
         "tiledb>=0.8.5",
-        "torch>=1.8.1",
-        "scikit-learn>=0.23.0",
-        "torchvision>=0.9.1",
-        "tensorflow>=2.5.0",
+        "tiledb-cloud>=0.7.11",
+        "matplotlib>=3.4.2",
```

Comment:
Why is matplotlib a dependency, let alone required?

Reply:
It's used in the examples. Maybe I can add extra cases.

Reply:
I'd say it's a bad form to force a dependency just for the examples; just `!pip install matplotlib` in the first code cell of the Jupyter notebook(s).